### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by kaizipaul**
In this PR:
-Added Linters
-Added HTML and CSS files 
-Added heading to HTML file
-Added styling to the heading using CSS 
-Linked the CSS file to the HTML file
-Updated the ReadMe file and .gitignore file

**Thank you for Reviewing my PR**

### CODE REVIEW

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hi @kaizipaul :wave: , 
### I'm Levy your PR reviewer,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct usage of git-flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly make sure to add a good PR title and PR description. It makes your repo looks more professional. :wink: 
   - Your PR
image
   - Example of good PR
image

- [x] Kindly make sure to write your commit message in imperative tense and capitalized. Please check this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand it clearly.

- [x] Kindly add descriptive README file to your project to make sure your looks repo more proffesional. Please take a step back and check [this template](https://github.com/microverseinc/readme-template) but remember you can always customize it and make more it professional and personal :sunglasses: 

- [x] Let's make sure to add `test.md` in `.gitignore`. `test.md`file will be create locally but will not be pushed to our github repository because it will be already added into our `gitignore` file. Kindly make sure you solve it as requested in the project requirements. :wink: 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kaizipaul** [comment]
In this PR:
-Added Linters
-Added HTML and CSS file
-Added heading to HTML file
-Added styling to CSS file
-Updated ReadMe and .gitignore file

**Comment by kaizipaul** [comment]
In this PR:
-Added Linters
-Added HTML and CSS files 
-Added heading to HTML file
-Added styling to the heading using CSS 
-Linked the CSS file to the HTML file
-Updated the ReadMe file and .gitignore file

**Thank you for Reviewing my PR**

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
...
+# Hello Microverse! Project
+>This project is made as a practise for HTML and CSS, and professional code writing.
+
+## Built with 
+- HTML
+- CSS
+
+## Author
+**Paul Kaizirege**
+- Github : [@kaizipaul]
```
Comment (HAMMAS-SALEEM) [inline README.md]: - [x] You didn't add all the fields from this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) in your readme file such as **Contributing**, **Show your support**, **Acknowledgement**. This is your readme file so please take it seriously :blush:
- [x] Also, add a license file in your repository you can add using this [template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) :blush:
- [x] Don't forget to add your name, current year, and app type in your MIT.md file after adding it in your project :+1:

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Ignore Mac system files
+.DS_store
+
```
Comment (HAMMAS-SALEEM) [inline .gitignore:3]: ### Wrong Place for Summary
summary

### RIght Place for Summary
right1

- [x] You should not add your summary in the message. Please follow the steps to update your summary :blush:
- **Step 1**

step1
- **Step 2**

step2

- **Step 3**
step3 (copy)

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# First of all Welcome to Microverse :tada: :bouquet:
# Required Changes :recycle:
almost-there

### Hey @kaizipaul :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Meaningful commits
:heavy_check_mark: Added .gitignore
:heavy_check_mark: Following Github Flow

Good job on implementing some of the changes requested by the previous reviewer, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional Suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. Otherwise reach out to me on slack which is more fast. You can reach out to me on slack by searching my name in **full-time-students** channel :+1:

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @kaizipaul  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**